src/CurrentMovie/CurrentMovie.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './CurrentMovie.css';
+
+const CurrentMovie = ({currentMovie, hideMovieDetails}) => {
+
+  return (
+    <div className='current-movie-details' onClick={() => hideMovieDetails()}>
+      <img src={currentMovie.backdrop_path} alt={`Still from ${currentMovie.title}`} />
+      <h2>{currentMovie.title}</h2>
+      <p>{`Released: ${new Date(currentMovie.release_date).toLocaleDateString({year: 'numeric', month: '2-digit', day: '2-digit'})} | Run Time: ${currentMovie.runtime} minutes`}</p>
```

Comment:
Consider moving logic like this to its own function to clear up the `return` statement.